docs/spot.md

```diff
@@ -0,0 +1,173 @@
+# Spot Setup
+
+## Installation
+
+### Basic
+
+#### 0. Requirements
```

Comment:
@Jdvakil can you help me update this documentation so we have a good "entry point" for Spot?